.github/workflows/push.yml

```
@@ -53,7 +53,7 @@ jobs:
           make -j4
           make install
       - name: 'Install node.js ${{ matrix.node-version }}'
-        uses: actions/setup-node@v2-beta
+        uses: actions/setup-node@v4.1.0
```

If you do not want Dependabot to be chatty...
```suggestion
        uses: actions/setup-node@v4
```